specs/protocol/block_header.md

```diff
@@ -0,0 +1,12 @@
+# Block Header
+
+## Application Header
+
+The application header is a network-agnostic block header. Different [networks](../network/index.md) may wrap the application header in a consensus header, depending on their consensus protocol.
+
+name                  | type       | description
+----------------------|------------|----------------------------------------------------------------------------------------------------------------------------------
+`txCount`             | `uint64`   | Number of [transaction](./tx_format.md#transaction)s in this block.
+`outputMessagesCount` | `uint64`   | Number of [output message](./tx_format.md#outputmessage)s in this block.
+`txRoot`              | `byte[32]` | [Merkle root](./cryptographic_primitives.md#binary-merkle-tree) of [transaction](./tx_format.md#transaction)s in this block.
+`outputMessagesRoot`  | `byte[32]` | [Merkle root](./cryptographic_primitives.md#binary-merkle-tree) of [output message](./tx_format.md#outputmessage)s in this block.
```

Comment:
todo: add `da_height` for blocks to signal the point of input message inclusion

Reply:
Fixed in [430780d](https://github.com/FuelLabs/fuel-specs/pull/405/commits/430780dfa10708c53e77a807681fc6ab7599c3f6)